HTML:CSS learning project/index.html

```diff
@@ -6,23 +6,25 @@
   <title>Road to web developer</title>
   <link rel="stylesheet" href="style.css">
 </head>
-<body>
-  <header>
+<body class="font">
+  <header class="main-header">
       <div class="container">
-        <nav>
-          <a href="#">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Services</a>
-          <a href="#">Contact Us</a>
+        <nav class="main-nav">
+          <div class="navigation">
+            <a href="#">Home</a>
+            <a href="#">About</a>
+            <a href="#">Products</a>
+            <a href="#">Services</a>
+            <a href="#">Contact Us</a>
+          </div>
         </nav>
     </div>
   </header>
   <main>
-    <img src="images/city.jpg">
+    <img src="images/city.jpg" class="city-image">
```

Comment:
Don't try to be specific when declaring classes. The main reason why we use classes is that we can reuse them later on different elements.

So in this case, the class is specifically for the city image but what's happen if the image is different and you want the same styles? (F.E car image). Then you need to apply .city-image class which doesn't make any sense semantically, so better refactor it to .image instead.